samples/nrf9160/aws_iot/src/main.c

```diff
@@ -20,6 +20,10 @@
 #include <cJSON.h>
 #include <cJSON_os.h>
 #include <zephyr/logging/log.h>
+#if IS_ENABLED(CONFIG_AWS_IOT_SAMPLE_DEVICE_ID_USE_HW_ID)
```

Comment:
```suggestion
#if defined(CONFIG_AWS_IOT_SAMPLE_DEVICE_ID_USE_HW_ID)
```

Reply:
In general we use `#if defined` for preprocessor statements. I think we should continue to stick to that.